traitsui/wx/image_slice.py

```diff
@@ -280,7 +280,7 @@ def _analyze_bitmap(self):
             else:
                 matches = [(dy / 2, 1)]
         elif n > self.stretch_rows:
-            matches.sort(lambda l, r: cmp(r[1], l[1]))
+            matches.sort(key=lambda x: x[1])
```

Comment:
It looks to me as though there's a reversal going on in the original code. Should we be using `reverse=True` or some such in this sorting operation? (Same comment below.)

Reply:
This whole section of code has the feel that it should either be re-written to be vectorized, or it should be cythonized.  Python for loops in analysis of images is Wrong.

Reply:
This is in image analysis? 🙀 I didn't look further than the lines in front of me.